src/update-labels-in-github.js

```diff
@@ -0,0 +1,166 @@
+const request = require('request-promise-native');
+const fs = require('fs');
+
+const validate = require('./config-validator.js');
+
+const program = require('commander');
+
+program
+    .usage('[options] <file>')
+    .option('-o, --org [org]', 'The name of the org.')
+    .option('-r, --repo [repo]', 'The name of the repo.');
+
+program.on('--help', function() {
+  console.log(`
+The following environment variables must be set:
+ - GIT_USER - the git user that owns the token specified in GIT_API_TOKEN
+ - GIT_API_TOKEN - the git API token used to access the GitHub API`);
+});
+
+program.parse(process.argv);
+
+if ((program.org && program.repo) || !(program.org || program.repo)) {
+  throw 'Either org or repo must be specified';
+}
+
+const configLocation = process.argv[process.argv.length - 1];
+const toUpdate = program.org ? { "org": program.org } : { "repo": program.repo };
+
+const user = process.env.GITHUB_USER;
```

Comment:
Small one - the readme and help says name the env vars `GIT_*` but the code is looking for `GITHUB_*`